### PR TITLE
eslint linter added

### PR DESCRIPTION
**Pull request by farhan787**
@jatinkatyal13 sir, please review it :-)

### CODE REVIEW

File: example/models/index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  dialect: "sqlite",
-  storage: "./database.sqlite"
+  dialect: 'sqlite',
+  storage: './database.sqlite'
 });
 
-const User = sequelize.define("user", {
-  // attributes
+const User = sequelize.define('user', {
+  // attributesg
```
Comment (jatinkatyal13) [inline example/models/index.js]: I guess this is a typo!

File: lib/util.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 const requireFromEnvironment = (variable, fallback) => {
   let value = process.env[variable];
 
   if (!value) {
     if (fallback) return fallback;
 
-    console.error("No $" + variable + " found in environment.");
-    console.log("You probably forgot to export the correct variables.");
+    console.error('No $' + variable + ' found in environment.');
```
Comment (jatinkatyal13) [inline lib/util.js]: add ${variable}

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "description": "Generic Base Controller for express to generate JSON API response",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/eslint/bin/eslint.js index.js example lib"
```
Comment (jatinkatyal13) [inline package.json]: add lint:fix script too with --fix flag

File: example/models/index.js (hunk 1 above)
Comment (farhan787) [reply example/models/index.js]: Yes, it's a typo.

File: lib/util.js (hunk 2 above)
Comment (farhan787) [reply lib/util.js]: I think prettier changed this.

File: package.json (hunk 3 above)
Comment (farhan787) [reply package.json]: Yeah, that should be there!

**Comment by jatinkatyal13** [comment]
LGTM 🚀

**Pull request merged**